anonymizer_methods.py

```diff
@@ -30,14 +35,16 @@ def Grep_DICOM_values(dicom_dir, dicom_fields):
     # Grep first dicom of the directory
     # TO DO: Need to check if file is dicom though, otherwise go to next one
     file_list = []
-    for f in os.listdir(dicom_dir):
-        file_list.append(f)
-    dicom_file = dicom_dir + os.path.sep + file_list[1]
+    for dirname, dirnames, filenames in os.walk(dicom_dir):
+        for filename in filenames:
+            dicom_file = os.path.join(dirname, filename)
+            break
+	
```

Comment:
This seems weird and I'm not sure what it's trying to do.

Presumably there's a reason to use os.walk instead of os.listdir, but the new logic doesn't look right to me. It has a for loop which always assigns a variable and breaks out unconditionally after the first iteration, and even then the assignment will get clobbered on subsequent iterations of the outer loop?